opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -406,12 +397,30 @@ mca_btl_tcp_endpoint_send_blocking(mca_btl_base_endpoint_t* btl_endpoint,
 
 static int mca_btl_tcp_endpoint_send_connect_ack(mca_btl_base_endpoint_t* btl_endpoint)
 {
-    /* send process identifier to remote endpoint */
+
     opal_process_name_t guid = opal_proc_local_get()->proc_name;
 
```

Comment:
Thanks for your comment. After discussing with @bwbarrett, here is my understanding. Adding a unique key to identify connection across processes in same job is a problem which should solve and currently i am not addressing that in this PR. I can iterate over in my next PR for it. Here we are just identifying where established connection is from MPI process.

Ya i agree adding 2 steps (send + recv of const string) is increasing overhead here. Do you have any suggestion here?